internal/vault/src/test/java/com/instana/vault/VaultServiceConfigTest.java

```diff
@@ -0,0 +1,79 @@
+package com.instana.vault;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class VaultServiceConfigTest {
+
+    private VaultServiceConfig config;
+
+    @BeforeEach
+    public void setUp() {
+        config = new VaultServiceConfig();
+    }
+
+    @Test
+    public void testGetConnectionURL() throws Exception {
+        String json = "{\"connection_url\": \"http://localhost:8200\"}";
```

Comment:
Please create a dummy json file and use it to convert to `VaultServiceConfig`.
Applicable to all the test cases.

Reply:
I have applied the suggested changes @BikramDebnath1, please have a look.